e2e/wdio/core/tests/grid-list.e2e-spec.ts

```diff
@@ -124,4 +137,249 @@ describe('Grid-list test suite', function() {
             gridListPage.checkRtlSwitch();
         });
     });
+
+    describe('Check visual regression', function() {
+
+        it('should check examples visual regression', () => {
+            gridListPage.saveExampleBaselineScreenshot();
+            expect(gridListPage.compareWithBaseline()).toBeLessThan(5);
+        });
+
+        it('verify buttons states', () => {
+            const buttonsLength = getElementArrayLength(gridListButtons);
+            for (let i = 0; i < buttonsLength; i++) {
+                scrollIntoView(gridListButtons, i);
+                checkElementHoverActiveStates(gridListButtons, buttonTag + i + '-', button, i);
+            }
+        });
+
+        it('verify toolbar states', () => {
+            scrollIntoView(gridListToolbar);
+            checkElementStates(gridListToolbar, toolbarTag, toolbar);
+        });
+
+        it('verify radio button states', () => {
+            const radioButtonLength = getElementArrayLength(gridListRadioButton);
+            for (let i = 0; i < radioButtonLength; i++) {
+                scrollIntoView(gridListRadioButton, i);
+                checkElementStates(gridListRadioButton, radioButtonTag + i + '-', radioButton, i);
+            }
+        });
+
+        it('verify checkbox states', () => {
+            const checkboxLength = getElementArrayLength(gridListCheckbox);
+            for (let i = 0; i < checkboxLength; i++) {
+                scrollIntoView(gridListCheckbox, i);
+                checkElementStates(gridListCheckbox, checkboxTag + i + '-', checkbox, i);
+            }
+        });
+
+        describe('Check none mode example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem);
+                checkElementStates(gridListItem, itemTag + '4-', item);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink);
+                checkElementStates(gridListLink, linkTag + '4-', link);
+            });
+        });
+
+        describe('Check single select left mode example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem, 13);
+                checkElementStates(gridListItem, itemTag + '5-', item, 13);
```

Comment:
I think you need to check 2 cards here because there are 2 types with 2 different style for click, etc.
![2021-04-09](https://user-images.githubusercontent.com/47522152/114139702-09f13680-9918-11eb-8afd-ff40ed2121a1.png)